src/main.rs

```diff
@@ -10,38 +10,32 @@ mod game;
 mod models;
 mod traits;
 
-use glutin_window::GlutinWindow;
-use opengl_graphics::{GlGraphics, OpenGL};
-use piston::event_loop::{Events, EventLoop};
-use piston::input::{Button, Event, Input, RenderEvent};
-use piston::window::WindowSettings;
+use piston_window::*;
+use opengl_graphics::GlGraphics;
 
-use drawing::Size;
-use game::Game;
 
-const OPEN_GL: OpenGL = OpenGL::V2_1;
+use game::Game;
 
 // Returns a result containing a GlutinWindow or an error if the window
 // settings are not supported
-fn try_create_window(samples: u8) -> Result<GlutinWindow, String> {
-    WindowSettings::new("Rocket", [1024, 600])
-        .exit_on_esc(true)
-        .opengl(OPEN_GL)
-        .samples(samples)
-        .build()
-}
-
 fn main() {
-    // Create a window with a sampling of 8 or fall back to 0
-    let window = try_create_window(8).or_else(|_| try_create_window(0)).unwrap();
 
-    let mut gl = GlGraphics::new(OPEN_GL);
+    let opengl = OpenGL::V3_2;
+
+    let mut game = Game::new(drawing::Size::new(1024.0, 600.0));
 
-    // The game object
-    let mut game = Game::new(Size::new(1024.0, 600.0));
+    let mut window: PistonWindow = WindowSettings::new("Rocket!", [1024, 600])
+        .opengl(opengl).exit_on_esc(true).build().unwrap();
```

Comment:
I think we should still try to create a window with `samples(8)` if it is supported by the hardware. The edges of the figures we draw become much smoother.